src/lib/modules/Base/Base.spec.js

```diff
@@ -0,0 +1,77 @@
+import BaseModule from '../Base';
```

Comment:
Module is `BaseModule` but file is `Base`. We should keep some conventions an name both the same way 😊 

Another question, why `BaseModule` ?